src/main/java/core/basesyntax/db/Storage.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax.db;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class Storage {
+    public static final Map<String, Integer> storage = new HashMap<>();
```

Comment:
According to the encapsulation principle, class fields should be private to hide the internal data and protect it from unauthorized access. Consider changing the access modifier of the `storage` map to private and provide a public method to interact with it if necessary.